src/js/manage.js

```diff
@@ -65,6 +85,19 @@ app.controller("manageController", ["$rootScope", "$scope", "$q", "pages", "acti
             }
             $scope.sortTypeGroups = type;
         };
+
+        // function for using datepicker in form for creating activities
+        $scope.datepicker = {open: false};
+        $scope.openDatePicker = function () {
+            $scope.datepicker.open = true;
+        };
+
+        $scope.dateOptions = {
+            formatYear: 'yy',
+            maxDate: new Date(2029, 5, 22), // maximum date for datepicker
```

Comment:
We define limits on dates in a few places now, and are inconsistent with the limits. Is it an idea to start doing this globally?

Reply:
jup good idea